Corona-Warn-App/src/main/java/de/rki/coronawarnapp/worker/DiagnosisTestResultRetrievalPeriodicWorker.kt

```diff
@@ -84,7 +78,8 @@ class DiagnosisTestResultRetrievalPeriodicWorker @AssistedInject constructor(
         return result
     }
 
-    private fun abortConditionsMet(): Boolean {
+    @VisibleForTesting
+    fun abortConditionsMet(currentMillis: Long): Boolean {
```

Comment:
If we mock time using `TimeStamper`, we might not need `VisibleForTesting`.

Reply:
done